packages/core/src/amazonqFeatureDev/util/files.ts

```diff
@@ -49,6 +60,17 @@ export async function prepareRepoData(
             zip.addLocalFile(file.fileUri.fsPath, zipFolderPath)
         }
 
+        ignoredExtensionMap.forEach(async (value, key) => {
+            await amznTelemetry.amazonq_bundleExtensionIgnored.run(async (bundleSpan) => {
```

Comment:
in this existing test for `prepareRepoData`, you can easily test the change in this PR using the `assertTelemetry` function.

the changes in this PR could easily have bugs, so that is worth doing

Reply:
alright, let me add tests